src/plugins/share/public/services/share_menu_manager.tsx

```diff
@@ -45,6 +46,7 @@ export class ShareMenuManager {
         const anonymousAccess = anonymousAccessServiceProvider?.();
         this.toggleShareContextMenu({
           ...options,
+          allowEmbed,
```

Comment:
As we are overriding it anyway, shouldn't we remove the `allowEmbed` from the `ShowShareMenuOptions`?

Reply:
thanks for highlighing this. This is an error and options.allowEmbed should be respected when kibana is not running in serverless. I have this logic to `allowEmbed: disableEmbed ? false : options.allowEmbed`